protocol/0028-GOVE-governance.md

```diff
@@ -215,10 +215,23 @@ Note the following key points from the market lifecycle spec:
 - A market enters a Pending status as soon as the proposal is Successful (before enactment)
 - A market usually enters Active status at the proposal's enactment date/time, but some conditions may delay this or cause the market to be Cancelled instead
 
-A proposal to create a market contains:
+A proposal to create a market contains
 
-1. a complete market specification as per the Market Framework (see spec) that describes the market to be created.
+1. a complete market specification as per the [Market Framework](./0001-MKTF-market_framework.md) that describes the market to be created.
 1. an enactment time that is at least the _minimum auction duration_ after the vote closing time (see [auction spec](./0026-AUCT-auctions.md))
+1. if the market is meant to be a _successor_ of a given market then it contains the marketID of the market it's succeeding (parent market), a parameter called `insurancePoolFraction` which is a decimal in `[0,1]` (i.e. it can be `0` or `1` or anything in between) and certain entries in the market proposal must be identical to those of the market it's succeeding OR the proposal should simply not contain the fields that cannot be changed.
+In a particular instrument, the settlement asset, margin asset, and `market.value.windowLength` must match.
+The parent market must be in one of `active` or `suspended` or `trading terminated` states or `settled` but with time since settlement less than or equal `network.liquidity.successorLaunchWindowLength`.
```

Comment:
```suggestion
The parent market must be in one of `active` or `suspended` or `trading terminated` states or `settled` but with time since settlement less than or equal `network.liquidity.successorLaunchWindowLength`. In particular the market cannot be `pending`.
```